docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='dec-table-create-proc']
+= Creating a speed violation guided decision table
+
+The following chapter instructs you how to create a speed violation decision table in {PRODUCT} using the Guided Decision Table wizard.
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Decision Table*, then enter:
+
+* *Guided Decision Table*: `SpeedViolationRule`
+* *Package*: `org.jboss.example.traffic_violations`
++
+
+. Select *Unique Hit* from the *Hit Policy* pull-down menu.
+. Select *Extended entry, values defined in table body* in the *Table format* section.
+. Click *Ok* to open the *Guided Decision Tables* designer.
++
+
+.Guided Decision Tables designer
+image::dec-table.png[]
+
+== Inserting Violation columns
```

Comment:
Ditto on intro statement, but even more so. Addressing what columns are/contain in GDTs is pretty important. See my comment in the Adding the Violation data object data fields section, about intro statements. And see my GDT doc for explanations, if you want: http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/

Reply:
done - not as detailed as yours, but suitable for this tutorial IMHO.